pg_search/src/postgres/customscan/aggregatescan/mod.rs

```diff
@@ -357,13 +357,78 @@ unsafe fn make_placeholder_func_expr(aggref: *mut pg_sys::Aggref) -> *mut pg_sys
     (*paradedb_funcexpr).funccollid = pg_sys::InvalidOid;
     (*paradedb_funcexpr).inputcollid = (*aggref).inputcollid;
     (*paradedb_funcexpr).location = (*aggref).location;
-    (*paradedb_funcexpr).args = PgList::<pg_sys::Node>::new().into_pg();
+
+    // Create a string argument with the aggregate function name for better EXPLAIN output
+    let agg_name = get_aggregate_name(aggref);
+    let agg_name_const = make_text_const(&agg_name);
+    let mut args = PgList::<pg_sys::Node>::new();
+    args.push(agg_name_const.cast());
+    (*paradedb_funcexpr).args = args.into_pg();
 
     paradedb_funcexpr
 }
 
+/// Get a human-readable name for the aggregate function
+unsafe fn get_aggregate_name(aggref: *mut pg_sys::Aggref) -> String {
+    // Try to get the function name from the catalog
+    let funcid = (*aggref).aggfnoid;
+    let proc_tuple =
+        pg_sys::SearchSysCache1(pg_sys::SysCacheIdentifier::PROCOID as _, funcid.into());
+
+    if !proc_tuple.is_null() {
+        let proc_form = pg_sys::GETSTRUCT(proc_tuple) as *mut pg_sys::FormData_pg_proc;
+        let name_data = &(*proc_form).proname;
+
+        let name_str = pgrx::name_data_to_str(name_data);
+
+        pg_sys::ReleaseSysCache(proc_tuple);
+
+        // Special case for pdb.agg() custom aggregate
+        if name_str == "agg" {
+            return "pdb.AGG".to_string();
+        }
```

Comment:
Nit: Could do this with the `aggfnoid` above I think.


```suggestion
        // Special case for pdb.agg() custom aggregate
        if name_str == "agg" {
            return "pdb.agg".to_string();
        }
```